tendermint/starport
# refactor(`network chain join`): separate message signer from address in gentx

Depends on https://github.com/tendermint/spn/pull/404

Update the code from the above request and makes the message signer independent from the address in the gentx
The `--from` is always used as the signer of the message
The address in the gentx is always used as the address for the genesis account to add

To test
Initialize the chain with an account and join it with another one
```
starport n --local chain publish https://github.com/lubtd/earth --no-check
starport n --local chain init 1 --from bob
starport n --local chain join 1 100000000stake --from alice
```
